### ISSUE TITLE
minishift status couldn't retrieve proper value of "DiskUsage" from remote host

### ISSUE THREAD

**Issue by amitkrout** [minishift/minishift#2478]
### General information

  * Minishift version: 
  * OS: Linux (CentOS Linux release 7.5.1804 (Core)) 
  * Hypervisor: generic

### Steps to reproduce

  1. Start a remote machine ```(CentOS)``` with all these [configuration](https://github.com/minishift/minishift/blob/master/docs/source/using/experimental-features.adoc#run-against-an-existing-machine)
  2. start minishift from the host using generic driver 
```
./minishift start --vm-driver generic --remote-ipaddress <ip address of the remote machine> --remote-ssh-user root --remote-ssh-key ~/.ssh/id_rsa
```
  3. Wait till minishift start is successful
```
[...]
Starting OpenShift using openshift/origin:v3.9.0 ...
OpenShift server started.

The server is accessible via web console at:
    https://10.70.45.10:8443

You are logged in as:
    User:     developer
    Password: <any value>

To login as administrator:
    oc login -u system:admin
```
  4. Check minishift status
```
$ ./minishift status
Minishift:  Running
Profile:    minishift
OpenShift:  Running (openshift v3.9.0+b8265d6-31)
DiskUsage:  ‘/mnt/?da1’: of df:
CacheUsage: 530 MB (used by oc binary, ISO or cached images)
```
### Expected
Should display % of disk is used

### Actual
DiskUsage:  ‘/mnt/?da1’: of df:

### Logs

You can start Minishift with `minishift start --show-libmachine-logs -v5` to collect logs.
Please consider posting this on http://gist.github.com/ and post the link in the issue.

**Comment by gbraad** [comment]
Remote disk setup is unknown to us. Maybe we should look for the root mount or if /var is mounted somewhere

**Comment by gbraad** [comment]
I have no issue with that... but I'd rather see a more general
approach/reusable method

**Comment by LalatenduMohanty** [comment]
> I have no issue with that... but I'd rather see a more general approach/reusable method

+1.  The output fields should be same. If we miss a field then from user point of view it is a bug. Hypothetically it can break any scripts if someone has automated it.

**Comment by gbraad** [comment]
If we can not produce a value; either write "Unknown", "N/A", "Not
available", or something similar. Better than leaving it completely out...

**Comment by LalatenduMohanty** [comment]
> "Unknown",

+1 for `unknown` or `N/A`

**Comment by LalatenduMohanty** [comment]
Resolved via #2486
